Neuron/Sections/Transaction/TransactionGasCostTableViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  TransactionGasCostTableViewController.swift
```

Comment:
Do not keep `TableView` in the class name (in case we change the view controller implementation detail in the future).